mlrun/api/api/endpoints/pipelines.py

```diff
@@ -118,6 +118,74 @@ async def create_pipeline(
     return response
 
 
+@router.get("/{run_id}")
```

Comment:
If you're already moving things around, maybe put the full path (from line 35) in each endpoint.

Reply:
this needs to be done on all endpoints once agreed 🤷🏼‍♂️